common/shared.c

```diff
@@ -512,6 +539,12 @@ int ft_client_connect(void)
 	if (ret)
 		return ret;
 
+	if (hints->caps & FI_RMA) {
+		ret = ft_set_rma_caps(fi, opts.rma_op);
+		if (ret)
+			return ret;
+	}
```

Comment:
Can this code segment move into ft_alloc_active_res()?  Both ft_client/server_connect call this beforehand.